hack/install-agent-rpm-ostree.sh.template

```diff
@@ -61,6 +60,15 @@ cat <<EOF > /etc/pki/consumer/cert.pem
 <CERT_PEM>
 EOF
 
+
+function get_client_id() {
+  if [ -f "/etc/device-credentials" ]; then
+    /usr/libexec/fdo/fdo-owner-tool dump-device-credential /etc/device-credentials | grep "Device GUID" | awk '{print $NF}'
```

Comment:
Is that command available by default in RHEL4Edge?

Reply:
yes, if the customizatoin-fdo is in there, it'll be in there.  this is why I checked the file.

Reply:
The other template doesn't have that `if` - does it meant that `fdo-owner-tool` is not available in other, dnf-supporting OSes?

Reply:
fdo-owner-tool is only availbe on rhel edge right now, fedora iot not yet.